examples/react-spa/src/Verification.tsx

```diff
@@ -89,7 +77,7 @@ export const Verification = (): JSX.Element => {
       getFlow(flowId).catch(createFlow)
       return
     }
-    createFlow()
+    createFlow().catch((error) => console.error(error))
```

Comment:
same here, i don't think createFlow returns the error

Reply:
Having `useCallback` in any of those `createFlow` is redundant, so will remove those promises altogether.